src/amr/messengers/mhd_hybrid_messenger_strategy.hpp

```diff
@@ -34,7 +34,8 @@ namespace amr
         }
 
         /**
-         * @brief allocate allocate the internal resources to the hybrid and MHD resourcesManagers
+         * @brief allocate allocate the internal resources to the hybrid and MHD
```

Comment:
double allocate

(I know it's formatting, but removing an "allocate" might help it use one line)